### PR TITLE
GH-50 Extract prefix to static variable in notifications.

### LINKED ISSUE #44: Template/prefix for notifcations.
image

### PR DESCRIPTION
**Pull request by VerniQ**

### CODE REVIEW

File: eternaleconomy-core/src/main/java/com/eternalcode/economy/config/implementation/messages/MessageConfig.java [hunk 1]
```diff
@@ -2,24 +2,27 @@
...
 import com.eternalcode.multification.notice.Notice;
 import eu.okaeri.configs.OkaeriConfig;
+import eu.okaeri.configs.annotation.Comment;
 
 @SuppressWarnings({"FieldMayBeFinal", "FieldCanBeLocal"})
 public class MessageConfig extends OkaeriConfig {
-
+    @Comment("Messages prefix")
+    public static String messagesPrefix =
+        "<b><gradient:#00FFA2:#34AE00>ECONOMY</gradient></b> <dark_gray>➤</dark_gray> ";
```
Comment (Rollczi) [inline eternaleconomy-core/src/main/java/com/eternalcode/economy/config/implementation/messages/MessageConfig.java:11]: Remove static

**Review by Rollczi — CHANGES_REQUESTED** [review]

**Review by CitralFlo — CHANGES_REQUESTED** [review]

**Review by vLuckyyy — CHANGES_REQUESTED** [review]
It should be done on the principle of internal placeholder, for example `{PREFIX}` can be used anywhere in the messages, this will allow a little more flexibility, currently you add everywhere, what if someone will have one message set to actionbar and the other to normal chat? I want to have a prefix in the chat and not in the actionbar, then he won't have a choice because you will add a prefix permanently.

**Review by Rollczi — CHANGES_REQUESTED** [review]
the solution used is based on incorrect assumptions. You can't expect that the prefix will magically be appended to the configuration, to understand this see what the generated config looks like and what happens if you change the prefix in file and reload it.


Tldr: this doesn't work

**Review by CitralFlo — CHANGES_REQUESTED** [review]
@Rollczi @vLuckyyy can it be extracted to notice creation so we do not have to setup {PREFIX} in every message?? Make it available by default for all messages?
